src/ir_Vestel.cpp

```diff
@@ -0,0 +1,335 @@
+// Copyright 2018 Erdem U. Altinyurt
+
+#include "ir_Vestel.h"
+#include <algorithm>
+#ifndef ARDUINO
+#include <string>
+#endif
+#include "IRremoteESP8266.h"
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+
+//                 VV     VV  EEEEEEE   SSSSS  TTTTTTTT  EEEEEEE  LL
+//                 VV     VV  EE       S          TT     EE       LL
+//                  VV   VV   EEEEE     SSSS      TT     EEEEE    LL
+//                   VV VV    EE            S     TT     EE       LL
+//                    VVV     EEEEEEE  SSSSS      TT     EEEEEEE  LLLLLLL
+
+// Vestel added by Erdem U. Altinyurt
+
+// Equipment it seems compatible with:
+//  * Vestel AC Model BIOX CXP-9 (9K BTU)
+//  * <Add models (A/C & remotes) you've gotten it working with here>
+
+// Ref:
+//   None. Totally reverse engineered.
+
+#if SEND_VESTEL_AC
+// Send a Vestel message
+//
+// Args:
+//   data:   Contents of the message to be sent.
+//   nbits:  Nr. of bits of data to be sent. Typically kVestelBits.
+//
+// Status: STABLE / Working.
+//
+void IRsend::sendVestelAC(uint64_t data, uint16_t nbits) {
```

Comment:
I thought **repeat** is some remotes mandatory spec.
Like if receiver expects same code twice to accept instead of CRC.

Reply:
Some protocols a `repeat` doesn't happen or makes little/no sense. That's probably the case here. However, it's part of the calling API is that all _real_ protocols support it.
e.g. Sony's protocol requires a min of 3 repeats. Sometimes it's used to simulate the button being held. Anyway, sendGeneric supports it, and you can use a default value in the header file so it can be invoked without it.